src/main/openapi/ds-storage-openapi_v1.yaml

```diff
@@ -77,6 +77,45 @@ paths:
                 format: int32
 
 
+  /origin/deleteRecordsForOrigin:
```

Comment:
Jeg har et problem med navngivning af endpoints. https://sbprojects.statsbiblioteket.dk/display/DT/API peger (i mangel af en decideret standard) på https://hackernoon.com/restful-api-designing-guidelines-the-best-practices-60e1d954e7c9 som et ideal for at definere API endpoints. Jeg vil anbefale en læsning - det tager 2-3 minutter.

Parametrene for den aktuelle metode er fine og matcher afsnit 6 på HackerNoon-siden, men `/origin/deleteRecordsForOrigin` clasher med afsnit 2.

Reply:
Method renamed here. But not for other methods, this will break backwards compatiblity.
Maybe a jira should be created for streamline all modules + change usage for depending modules using the client.